Makefile

```diff
@@ -15,7 +15,7 @@
 # Bump these on release - and please check ISO_VERSION for correctness.
 VERSION_MAJOR ?= 1
 VERSION_MINOR ?= 18
-VERSION_BUILD ?= 0
+VERSION_BUILD ?= 1
```

Comment:
Bump the ISO version as well I believe

Reply:
Actually, with this comment I'm not sure...
```
# Default to .0 for higher cache hit rates, as build increments typically don't require new ISO versions
```

Reply:
did we change anything in iso?

Reply:
I don't think so, I'm just not sure if we duplicate the v1.18.0 ISO and rename it to v1.18.1 or not

Reply:
Based on the comment I don't think we do

Reply:
we didnt change ISO so no need to do that